launcher/src/main/java/org/apache/spark/launcher/SparkLauncher.java

```diff
@@ -64,6 +64,10 @@
   /** Configuration key for the number of executor CPU cores. */
   public static final String EXECUTOR_CORES = "spark.executor.cores";
 
+  public static final String PYSPARK_DRIVER_PYTHON = "spark.pyspark.driver.python";
```

Comment:
Do you mean to define them in 2 places (`package.scala` & `SparkLauncher`) ? This may cause some inconsistency in future although it is very unlikely to change them.